test/unit/lib/plugins/aws/deploy/lib/createStack.test.js

```diff
@@ -126,11 +126,11 @@ describe('createStack', () => {
     it('should throw error if invalid stack name', () => {
       sandbox.stub(awsDeploy, 'create').resolves();
       sandbox.stub(awsDeploy.provider, 'request').resolves();
-      awsDeploy.serverless.service.service = 'service-name'.repeat(100);
+      awsDeploy.serverless.service.provider.stackName = 'service-name'.repeat(100);
```

Comment:
Is there a reason to change this test setup here?

Reply:
Huh, I'm not sure how or why that snuck in there.... I did perform a rebase, but to answer your question - no, there's no reason that that should be there.